app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<style>
+  .title {
+    text-align: center;
+  }
+  .new_user {
+    text-align: center;
+  }
+  .all_users {
+    text-align: center;
+  }
+  .existing_users {
+    font-weight: bold;
+  }
+</style>
```

Comment:
Front end! Looks good, I wonder if this could live in layouts or somewhere we could share styling.